ipapython/ipa_log_manager.py

```diff
@@ -197,6 +250,12 @@ def convert_log_level(value):
     return level
 
 
+# Add info_cmd to logger and root logger
+if not hasattr(logging.getLoggerClass(), "info_cmd"):
+    logging.getLoggerClass().info_cmd = log_info_cmd
```

Comment:
You are suppose to use [logging.setLoggerClass](https://docs.python.org/3/library/logging.html#logging.setLoggerClass) to define your own logger subclass.

Reply:
This is not needed with the latest version anymore.